### PR TITLE
refactor(accordion): split component - INNO-1562

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Split accordion and accordion2 into 2 components.

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit d32fb3ea1d0a304f0bb8de727b8839428ecec8bc

https://deploy-preview-1194--europa-component-library.netlify.com

File: src/systems/ec/specs/components/accordion2/demo/data.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// Simple content for demo
+module.exports = {
+  items: [
+    {
+      id: 'accordion-example',
+      level: 3,
+      toggle: {
+        label:
+          'Delivery of last pending proposals, a common Destiny of unity, the hour of European Democracy',
+        iconShape: 'general--digital',
```
Comment (yhuard) [inline src/systems/ec/specs/components/accordion2/demo/data.js]: is `iconShape` actually used?

File: src/systems/eu/implementations/react/components/accordion/stories/Index.jsx [hunk 2]
```diff
@@ -39,6 +40,40 @@ storiesOf('Components|Accordion', module)
       {story()}
     </StoryWrapper>
   ))
+  .addDecorator(withCssResources)
+  .addParameters({
+    cssresources: [
+      {
+        id: 'ecl-legacy-screen',
+        code: `<link rel="stylesheet" type="text/css" href="./styles/ecl-ec-preset-legacy-website.css" />`,
```
Comment (yhuard) [inline src/systems/eu/implementations/react/components/accordion/stories/Index.jsx]:
```suggestion
        code: `<link rel="stylesheet" type="text/css" href="./styles/ecl-eu-preset-legacy-website.css" />`,
```

File: src/systems/eu/implementations/react/components/accordion/stories/Index.jsx [hunk 3]
```diff
@@ -39,6 +40,40 @@ storiesOf('Components|Accordion', module)
...
+        picked: true,
+      },
+      {
+        id: 'ecl-legacy-print',
+        code: `<link rel="stylesheet" type="text/css" href="./styles/ecl-ec-preset-legacy-website-print.css" />`,
```
Comment (yhuard) [inline src/systems/eu/implementations/react/components/accordion/stories/Index.jsx]:
```suggestion
        code: `<link rel="stylesheet" type="text/css" href="./styles/ecl-eu-preset-legacy-website-print.css" />`,
```

**Review by yhuard — CHANGES_REQUESTED** [review]

File: src/systems/ec/specs/components/accordion2/demo/data.js (hunk 1 above)
Comment (emeryro) [reply src/systems/ec/specs/components/accordion2/demo/data.js]: not in accordion2 indeed, good catch

**Review by yhuard — APPROVED** [review]

**Pull request merged**